core-bundle/README.md

```diff
@@ -150,6 +161,8 @@ security:
         - { path: ^/contao/login$, roles: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: ^/contao/logout$, roles: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: ^/contao(/|$), roles: ROLE_USER }
+        - { path: ^/_contao/two-factor, roles: [IS_AUTHENTICATED_2FA_IN_PROGRESS, ROLE_MEMBER] }
+        - { path: ^, roles: [IS_AUTHENTICATED_2FA_IN_PROGRESS, IS_AUTHENTICATED_ANONYMOUSLY] }
```

Comment:
What kind of`path` is this? I've never seen such a config. I guess it should be `^/`, shouldn't it?

Reply:
Could probably be. Just to clarify, we need to allow any path with `IS_AUTHENTICATED_2FA_IN_PROGRESS`, otherwise the two-factor implementation will disallow access and redirect to the form endpoint, which does not exist in the front end: https://github.com/scheb/two-factor-bundle/blob/master/Security/Authorization/TwoFactorAccessDecider.php#L48
We might want to override this implementation, but we should do that in a secondary PR.

Reply:
Then that has to be `^/`, yes.

Reply:
Changed in c63b117823c61cc666ddd378796500622f2861f9